pkg/cmd/tasks/execute/execute.go

```diff
@@ -238,3 +244,17 @@ func slugFromScript(file string) (string, error) {
 
 	return slug, nil
 }
+
+type notDeployedError struct {
+	task string
+}
+
+// Error implementation.
+func (err notDeployedError) Error() string {
+	return fmt.Sprintf("task %s was not deployed", err.task)
+}
+
+// ExplainError implementation.
+func (err notDeployedError) ExplainError() string {
+	return fmt.Sprintf("to deploy the task:\n\tairplane deploy %s", err.task)
```

Comment:
Are we using `\t` everywhere? Can we just do two spaces for now, to match https://github.com/airplanedev/cli/blob/main/pkg/cmd/tasks/initcmd/init.go#L139

```suggestion
	return fmt.Sprintf("To deploy the task:\n  airplane deploy %s", err.task)
```

Reply:
Same as the comment above.

Reply:
just to avoid lint errors btw, also cleanliness

Reply:
huh interesting (https://github.com/golang/go/wiki/CodeReviewComments#error-strings for reference) - cc @colinking @francescaleung on this, so we're consistent.

(minor note - was the `\t` intentionally left behind? can we use spaces instead?)

Reply:
ah yikes, yeah I'll fix it right away.